conftest.py

```diff
@@ -62,10 +63,8 @@ def setup_all(request):
     print(
         f"sender: {wallet.key}, {wallet.address}, {wallet.password}, {wallet.keysStr()}"
     )
-    print(
-        f"sender balance: {Web3Helper.from_wei(Web3Helper.get_ether_balance(wallet.address))}"
-    )
-    assert Web3Helper.from_wei(Web3Helper.get_ether_balance(wallet.address)) > 10
+    print(f"sender balance: {from_wei(get_ether_balance(wallet.address))}")
+    assert from_wei(get_ether_balance(wallet.address)) > 10
```

Comment:
I forgot about this when I approved. Please add messages to all asserts, per #283. It is in progress on another PR to add them retrospectively, so let's add them in all the new PRs as well.

Reply:
Actually, I needed the web3 helper changes in #309, so I added the asserts. Hope you don't mind! 😄